eahub/profiles/models.py

```diff
@@ -530,9 +531,23 @@ def convert_to_row(self, field_names):
         return values
 
     def _format_enum_array_for_searching(
-        self, array: List[enum.Enum], enum_cls: enum.Enum
+        self,
+        enum_values_list: List[Union[enum.Enum, str, int]],
```

Comment:
why do we need this catch all of enums, strings and ints here? All usages of the function seem to be passing enums, not str or int

Reply:
good point, it's supposed to, but in debugger I noticed that sometimes I a enum value as an int `1` and sometimes as a string `'1'` ¯\_(ツ)_/¯

I think we're thinking about migrating to Model-based FKs, right?

Reply:
Super weird. Was this the cause of the bug then? In any case, I'm fine with leaving it like this. 
I'm not aware that we had any discussion around FKs, maybe I missed something?

Reply:
The cause might be the incorrect enum implementation - I guess the enum package that we're using is outdated since python3.7 introduced the official enum specification, which is significantly different, but in a better way.

> I'm not aware that we had any discussion around FKs, maybe I missed something?

I'm pretty sure we discussed it with you as well on gdrive or github - in short in django the best practice for this use case are FKs or M2M models with enabled native `list_filter` 👍

Reply:
ie similar to how django handles groups & permissions:

![image](https://user-images.githubusercontent.com/36416236/102695704-ae4c5180-4231-11eb-9d94-8590a6eac754.png)